src/tests/ftest/util/pool_security_test_base.py

```diff
@@ -323,7 +324,7 @@ def setup_container_acl_and_permission(
             user_type, user_name, perm_type, permission)
         result = self.update_container_acl(
             secTestBase.acl_entry(user_type, user_name, permission))
-        if result.stderr is not "":
+        if result.stderr_text is not "":
```

Comment:
(pylint-literal-comparison) Comparison to literal

Reply:
Fixed